bin/setup_integration_test.sh

```diff
@@ -6,18 +6,10 @@
 
 # Sets up S3 buckets for integration test.
 
-if [ -z "${SUBMITTER_S3_BUCKET}" ]; then
-    echo "SUBMITTER_S3_BUCKET is not set."
-    exit 1;
-fi
+set -euo pipefail
 
-if [ -z "${CRASHSTORAGE_BUCKET_NAME}" ]; then
-    echo "CRASHSTORAGE_BUCKET_NAME is not set."
-    exit 1;
-fi
```

Comment:
These just checked to make sure the variables were defined. But we're doing that with the `set -euo pipefail` so we can nix these.